setup.py

```diff
@@ -37,5 +37,5 @@
         "Operating System :: OS Independent",
         "Framework :: Flask",
     ],
-    python_requires=">=3.6,<3.9",
+    python_requires=">=3.7,<=3.8",
```

Comment:
why no support for 3.9?

Reply:
I wasn't able to find a `<=` example in the [pep docs](https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires), are you sure it will work?

Reply:
No, i'm not sure. But it doesn't matter so u've reverted that part.